css/src/editor.scss

```diff
@@ -158,6 +158,7 @@ div[class^="wp-block-block-lab-"] {
       margin: 1em 0 0;
 
       .components-base-control__label {
+        display: block;
```

Comment:
This fixes an issue with a few fields, like Color and Select:

<img width="615" alt="above-here" src="https://user-images.githubusercontent.com/4063887/68078026-1d4bba00-fd94-11e9-91c1-c08af1743b45.png">